modules/nf-core/sigprofiler/extractor/main.nf

```diff
@@ -0,0 +1,56 @@
+process SIGPROFILER_EXTRACTOR {
+    tag "$meta.id"
+    label 'process_single'
+
+    // TODO nf-core: List required Conda package(s). Software MUST be pinned to channel (i.e. "bioconda"), version (i.e. "1.10").
+    // TODO nf-core: See section in main README for further information regarding finding and adding container addresses to the section below.
+    conda "${moduleDir}/environment.yml"
+    container "sigprofilerextractor"
```

Comment:
The container must be hosted on Quay (quay.io) to work here, see some other modules (really, any) to see the correct way to link the containers